pom.xml

```diff
@@ -193,7 +167,9 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
+        <version>${maven-enforcer-plugin.version}</version>
```

Comment:
Isn't this version already defined in the parent pom, then we do not need it here?

Reply:
I am just being consistent with your other code in this POM file that already contains redundant versions. If you think the redundancy should be eliminated, remove it from your existing code in a separate PR and then I will follow suit in this PR.